README.md

```diff
@@ -11,316 +11,264 @@
    <img src="docs/source/_static/images/general/model_evaluation_suite.gif" width="800">
 </p>
 
+</details>
 
-Note:
-
-- Results can be [displayed](https://docs.deepchecks.com/stable/user-guide/general/showing_results.html) in various manners, or [exported](https://docs.deepchecks.com/stable/user-guide/general/export_save_results.html) to an html report, saved as JSON, or integrated with other tools (e.g. wandb).
-- Other suites that run only on the data (``data_integrity``, ``train_test_validation``) don't require a model as part of the input.
-
-See the [full code tutorials
-here](
-   https://docs.deepchecks.com/stable/user-guide/tabular/auto_quickstarts/index.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out).
-
-
-In the following section you can see an example of how the output of a single check without a condition may look.
-
-### Running a Check
+<details open>
+   <summary>
+      <h4>
+         Deepchecks Monitoring Quickstart
+      </h4>
+   </summary>
 
-To run a specific single check, all you need to do is import it and then
-to run it with the required (check-dependent) input parameters. More
-details about the existing checks and the parameters they can receive
-can be found in our [API
-Reference](https://docs.deepchecks.com/stable/api/index.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=running_a_check).
-
-```python
-from deepchecks.tabular.checks import FeatureDrift
-import pandas as pd
-
-train_df = pd.read_csv('train_data.csv')
-test_df = pd.read_csv('test_data.csv')
-# Initialize and run desired check
-FeatureDrift().run(train_df, test_df)
-```
-
-Will produce output of the type:
-
->   <h4>Train Test Drift</h4>
->  <p>The Drift score is a measure for the difference between two distributions,
->   in this check - the test and train distributions. <br>
->   The check shows the drift score and distributions for the features,
->   sorted by feature importance and showing only the top 5 features, according to feature importance.
->   If available, the plot titles also show the feature importance (FI) rank.</p>
->   <p align="left">
->      <img src="docs/source/_static/images/general/train-test-drift-output.png">
->   </p>
-
-
-## 🙋🏼  When Should You Use Deepchecks?
-
-While you’re in the research phase, and want to validate your data, find potential methodological problems, 
-and/or validate your model and evaluate it.
+Jump straight over to the 
+[monitoring quickstart docs](https://docs.deepchecks.com/monitoring/stable/user-guide/tabular/auto_quickstarts/plot_quickstart.html)
+to have it up and running on your data.
 
+You'll then be able to see the checks results over time, set alerts, and interact
+with the dynamic deepchecks UI that looks like this:
 
 <p align="center">
-   <img src="/docs/source/_static/images/general/pipeline_when_to_validate.svg">
+   <img src="docs/source/_static/images/general/monitoring-app-ui.gif" width="800">
```

Comment:
Not rendered in git, dunno if it works when building docs.

Reply:
Previous gif did render so a bit suspicious

Reply:
try refreshing? (maybe internet connection cause it works for me)